masterlist.yaml

```
@@ -23821,3 +23821,11 @@ plugins:
     clean:
       - crc: 0x2358D3DD
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
+
+  - name: 'FGMoreAdventures.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/40029/' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0xDC20C34C
+        util: '[SSEEdit v4.0.4]'
+        itm: 2
```

The link in the `util` somehow got excised. Also, since the cleaning data is the same for xEdit v4.0.4b, might as well bump the version.

```suggestion
    dirty:
      - <<: *quickClean
        crc: 0xDC20C34C
        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 2
```